app/boards/shields/zmk_uno/zmk_uno_split.dtsi

```diff
@@ -31,18 +33,57 @@
 
     split_direct_matrix_transform: split_direct_matrix_transform {
         compatible = "zmk,matrix-transform";
-        rows = <3>;
+        rows = <2>;
         columns = <4>;
 
         map = <
         RC(0,0) RC(0,1)
         RC(0,2) RC(0,3)
 
-        RC(2,0) RC(2,1)
-        RC(2,2) RC(2,3)
+        RC(1,0) RC(1,1)
+        RC(1,2) RC(1,3)
         >;
     };
 
+    matrix_physical_layout: matrix_physical_layout {
+        compatible = "zmk,physical-layout";
+        display-name = "Matrix Layout";
+
+        kscan = <&kscan_matrix>;
+        transform = <&split_matrix_transform>;
+
+        keys
+            = <&key_physical_attrs 100 100   0   0 0 0 0>
+            , <&key_physical_attrs 100 100 100   0 0 0 0>
+            , <&key_physical_attrs 100 100   0 100 0 0 0>
+            , <&key_physical_attrs 100 100 100 100 0 0 0>
+            , <&key_physical_attrs 100 100 200   0 0 0 0>
+            , <&key_physical_attrs 100 100 300   0 0 0 0>
+            , <&key_physical_attrs 100 100 200 100 0 0 0>
+            , <&key_physical_attrs 100 100 300 100 0 0 0>
+            ;
```

Comment:
I don't know how much you care about these at this stage, but this physical layout spec doesn't match the transform/keymap shape:

![image](https://github.com/zmkfirmware/zmk/assets/7876996/1a1ac9b4-80ca-40ae-b5d5-e1fc560bc496)

To fix that we'd either have to move the right 4 keys below the left in `keys`, or change matrix-transform+keymap order and reorder items in `keys`.

Reply:
Good catch. Fixed the physical layout to match the keymap/transform.

Reply:
The direct wire one looks good now, but the matrix layout one is still side by side: 
![image](https://github.com/zmkfirmware/zmk/assets/7876996/ad803cf9-c9ab-468d-aff5-05cdc9badf60)
I think it could be the same as the direct wire one, right?

Reply:
Oh indeed. Fixed. Thanks.